main.nf

```diff
@@ -1241,6 +1282,7 @@ process GET_SOFTWARE_VERSIONS {
     echo $workflow.manifest.version > v_pipeline.txt
     echo $workflow.nextflow.version > v_nextflow.txt
     qcat --version &> v_qcat.txt
+    NanoLyse --version &> v_nanolyse.txt
```

Comment:
Need to add this here too https://github.com/nf-core/nanoseq/blob/ad5b2bb7a37a6edb9f61603636bd26e43259f58b/bin/scrape_software_versions.py#L11

Reply:
I have added a similar line for the NanoLyse process.